explore/src/main/scala/explore/tabs/TargetTabContents.scala

```diff
@@ -319,20 +320,21 @@ object TargetTabContents {
                                 Constants.InitialTreeWidth.toInt
           )
           .attempt
-          .map {
+          .flatMap {
             case Right(w) =>
-              Callback.log(s"Set $w") *>
-                panels
-                  .zoom(TwoPanelState.treeWidth[ObsIdSet])
-                  .set(w)
-            case Left(u)  => Callback.log(u.toString)
+              panels
+                .zoom(TwoPanelState.treeWidth[ObsIdSet])
+                .set(w)
+                .to[IO]
+            case Left(_)  => IO.unit
           }
-          .runAsyncAndForget
+          .runAsync
       }
-      .renderWithReuse { (props, tps, opts) =>
+      .useResizeDetector(UseResizeDetectorProps())
```

Comment:
Can we call `.useResizeDetector()` ?

Reply:
I guess we should be able to add a default. let me check it

Reply:
Ah ok, I saw a `()` version defined, but it's only at the low level.